tsl/test/sql/merge_chunks.sql

```diff
@@ -375,3 +376,40 @@ select * from compression_size_fraction;
 select count(*), sum(device), round(sum(temp)::numeric, 4) from mergeme;
 select * from partitions;
 select * from chunk_info;
+rollback;
+
+----------------------------
+-- Test concurrent merges --
+----------------------------
+
+create view chunks_being_merged as
+select
+    chunk_relid,
+    n as new_chunk_id,
+    (select count(indexrelid::regclass) from pg_index where indrelid=new_relid) as num_new_indexes
+from (select *, dense_rank() over (order by new_relid) as n from _timescaledb_catalog.chunk_rewrite) cr;
+
+-- Concurrent merge cannot run in a transaction block
+\set ON_ERROR_STOP 0
+begin;
+call merge_chunks_concurrently(ARRAY['_timescaledb_internal._hyper_1_1_chunk', '_timescaledb_internal._hyper_1_4_chunk','_timescaledb_internal._hyper_1_5_chunk', '_timescaledb_internal._hyper_1_12_chunk']);
+rollback;
+
+select debug_waitpoint_enable('merge_chunks_fail');
+call merge_chunks_concurrently(ARRAY['_timescaledb_internal._hyper_1_1_chunk', '_timescaledb_internal._hyper_1_4_chunk','_timescaledb_internal._hyper_1_5_chunk', '_timescaledb_internal._hyper_1_12_chunk']);
+\set ON_ERROR_STOP 1
```

Comment:
Do we want a successful merge_chunks_concurrently() test case as well?

Reply:
This is mostly tested in isolation tests (different test file) since the "concurrent" aspect of the command cannot be tested here. But it might make sense to have at least one successful merge here too, so I'll add it.